test_create_geometry_3D.py

```diff
@@ -0,0 +1,29 @@
+import gmsh
+import sys
+import pytest
+from create_geometry_3D import make_geometry_3D, create_point_pairs
+from create_geometry_3D import extract_mesh_data
+
+
+def test_make_geometry():
```

Comment:
this test can be put in the same file as test_create_geometry_2D, and then the filename can be changed to test_create_geometry.py